.ci/azure-pipelines/docs-pipeline.yaml

```diff
@@ -0,0 +1,50 @@
+trigger:
+  paths:
+    include:
+    - doc
+    - README.md
+    - CHANGES.md
+    - CONTRIBUTING.md
```

Comment:
Why? :) No need to include these 3 md files

Reply:
I apologize for that I've not explain my purpose well.

The expected behavior is:
- Docs change: only (push or PR) trigger docs pipeline(contains `formatting` now)
- Code change: only (push or PR) trigger main pipeline, and (pipeline) trigger docs pipeline, which means docs pipeline won't be triggered instantly only after `build_gcc` stage complete nicely.

It's a bit hard to implement flow control between different pipelines, so I use pipeline trigger to do this:
> The tutorials stage should be run after the documentation stage, only if the ubuntu build CI goes green.

To prevent the docs-pipeline running twice in code changes (the push&PR trigger, pipeline trigger), I restrict the push&PR trigger in docs-pipeline by adding `include` rules.

Reply:
Thanks for the explanation, but my point was that these 3 lines are not part of the documentation either, so no need to trigger the documentation pipeline

Reply:
Ah...it's just my carelessness, thank you so much!!